### DISCUSSION TITLE
[Possible Question #1] Q&A area about Wikimedia Hackathon 2023 blogging

### DISCUSSION THREAD

**Discussion by wmhack2023** [wmhack2023/wmhack2023.github.io#7] [Q&A] [answered]
Go through the Q&A regarding blogging about Wikimedia Hackathon 2023 or start a new question thread if needed

**Comment by p20magi** [comment]
Where should we upload photos from the Hackathon?

**Reply by riggas-ionio** [reply, marked as answer]
Photos should be uploaded in Wikimedia Commons under the category Wikimedia_Hackathon_Athens_2023:
https://commons.wikimedia.org/wiki/Category:Wikimedia_Hackathon_Athens_2023
